### ISSUE TITLE
Rename things in Sanity Studio

### ISSUE THREAD

**Issue by vnys** [equinor/energyvision#368]
To make the editors feel more like home in the Studio I propose the following name changes:

- Topic content → Page Builder
- Topic Routes → Routes
- Top Level Pages → Landing Pages

The first two are names we used in the POC, the third one is what @nicholas-haggerty calls Top Level Pages if I’m not mistaken.

**Comment by vnys** [comment]
Att: @BirteThornquist, @SvSven, @wenche

**Comment by SvSven** [comment]
I think we can turn this into a larger issue actually, and incorporate it into updating the labels and descriptions for menu items and input fields.
The user testing sessions we had this week have given some good feedback specifically on that.

My impression from that is that the difference between "Topic content" and "Topic routes" is actually understood, but we'll have to wait for the summary and details once @BirteThornquist is done. We should definitely get more feedback on these things from users and base the naming on that instead of on what we think is appropriate however.

**Comment by nicholas-haggerty** [comment]
It was good to see the examples sent over from Sanity the other day of the set up of a couple client structures. Very logical naming policy for the estate agents one and easy journey into the content you would want to change/update. A good editor has been involved there!
Agree @sven the Topic pages and routes seemed to be understood during the tests, so It might just be me that struggles with these terms. But overall some very common sticking points that we can easily rectify.

**Comment by SvSven** [comment]
Putting this "on hold" pending the results/summary of the user tests from @BirteThornquist 
Once we have those we should plan a meeting or workshop to get a consensus.Need to put good thought into the naming based on feedback before we start changing things.

**Comment by nilsml** [comment]
@BirteThornquist to schedule workshop
